source/codegen/metadata/niscope/attributes.py

```diff
@@ -1,2556 +1,1558 @@
     },
     1150354: {
-        'access': 'read-write',
-        'channel_based': False,
-        'codegen_method': 'public',
-        'documentation': {
-            'description': '\nSpecifies whether the digitizer writes data to onboard memory when a peer-to-peer endpoint is enabled.\nDefault Value: VI_FALSE\n',
-            'note': 'This attribute can be used only with high-speed digitizers that support peer-to-peer streaming.'
-        },
-        'lv_property': 'Peer-to-Peer:Onboard Memory Enabled',
+        'codegen_method': 'public',
+        'grpc_type': 'bool',
         'name': 'P2P_ONBOARD_MEMORY_ENABLED',
         'resettable': True,
         'type': 'ViBoolean'
     },
     1150366: {
-        'access': 'read-write',
-        'attribute_class': 'AttributeViReal64TimeDeltaSeconds',
-        'channel_based': False,
-        'documentation': {
-            'description': '\nEnd of Record to Advance Trigger Holdoff is the length of time (in\nseconds) that a device waits between the completion of one record and\nthe acquisition of pre-trigger samples for the next record. During this\ntime, the acquisition engine state delays the transition to the Wait for\nAdvance Trigger state, and will not store samples in onboard memory,\naccept an Advance Trigger, or trigger on the input signal..\n**Supported Devices**: NI 5185/5186\n'
-        },
-        'lv_property': 'Triggering:End of Record to Advance Trigger Holdoff',
+        'codegen_method': 'public',
+        'grpc_type': 'double',
         'name': 'END_OF_RECORD_TO_ADVANCE_TRIGGER_HOLDOFF',
         'resettable': True,
-        'type': 'ViReal64',
-        'type_in_documentation': 'hightime.timedelta, datetime.timedelta, or float in seconds'
+        'type': 'ViReal64'
     },
     1150367: {
-        'access': 'read-write',
-        'channel_based': False,
-        'documentation': {
-            'description': '\nIf NISCOPE_ATTR_SAMP_CLK_TIMEBASE_SRC is an external source, this property specifies the ratio between the NISCOPE_ATTR_SAMP_CLK_TIMEBASE_RATE and the actual sample rate, which can be higher. This property can be used in conjunction with NISCOPE_ATTR_SAMP_CLK_TIMEBASE_DIV.\nSome devices use multiple ADCs to sample the same channel at an effective sample rate that is greater than the specified clock rate. When providing an external sample clock use this property to indicate when you want a higher sample rate. Valid values for this property vary by device and current configuration.\n\n**Related topics:**\n`Sample Clock <digitizers.chm::/Sample_Clock.html>`__\n'
-        },
+        'codegen_method': 'public',
+        'grpc_type': 'sint32',
         'name': 'SAMP_CLK_TIMEBASE_MULT',
-        'python_name': 'sample_clock_timebase_multiplier',
         'resettable': True,
         'type': 'ViInt32'
     },
     1150373: {
-        'access': 'read-write',
-        'channel_based': False,
         'codegen_method': 'public',
-        'enum': 'P2PStreamRelativeTo',
```

Comment:
Missing 'enum' tag for this attribute. You can look [here](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/358221?_a=files&path=/src/nidmm/nidmmBaseDistribution/nidmmAPIMetadata/source/nidmmAPIMetadata/grpc_device/attributes/attributes.hapigen) for an example of how to add these tags.